### PR TITLE
Layout & Template Added for Dashboard Pages

### PR DESCRIPTION
**Pull request by 13x54n**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -17,6 +17,8 @@
     "react-dom": "^18.2.0",
     "react-drag-drop-files": "^2.3.10",
     "react-router-dom": "^6.20.0",
+    "react-rte": "^0.16.5",
+    "react-tag-input-component": "^2.0.2",
```
Comment (13x54n) [inline frontend/package.json:21]: React Tags and Rich Text Editor

File: frontend/src/components/Dashboard/DashboardRouter.jsx [hunk 2]
```diff
@@ -19,6 +20,8 @@ export default function DashboardRouter({ activePage }) {
       return <Layout component={DHome} />;
     case "orders":
       return <Layout component={DOrders} />;
+    case "categories":
+      return <Layout component={DCategory} />;
```
Comment (13x54n) [inline frontend/src/components/Dashboard/DashboardRouter.jsx:24]: Added Category Page

File: frontend/src/components/Dashboard/NewProduct.jsx
Comment (13x54n) [inline frontend/src/components/Dashboard/NewProduct.jsx:1]: New Product Upload Form, Need to work on backend.

File: frontend/src/mocks/Sidebar.json
Comment (13x54n) [inline frontend/src/mocks/Sidebar.json:1]: Sidebar Updated

File: frontend/src/pages/Dashboard.jsx [hunk 3]
```diff
@@ -66,7 +66,7 @@ export default function Dashboard() {
           <div>Zetsy</div>
           <div className="flex flex-row gap-2">
             <button
-              onClick={() => window.open("https://zetsy.store", "_blank")}
+              onClick={() => window.open(`https://${selected.storeUrl}`, "_blank")}
```
Comment (13x54n) [inline frontend/src/pages/Dashboard.jsx:69]: Dynamic Domain Routing must be done.

**Review by 13x54n — COMMENTED** [review]
Reviewed on my side.

File: frontend/src/pages/Dashboard.jsx (hunk 3 above)
Comment (surajgaire14) [reply frontend/src/pages/Dashboard.jsx:69]: I do not know more about dynamic routing and sub-domains, other than that it seems pretty good.

**Pull request merged**